KMB/make_KMB_info.py

```diff
@@ -670,7 +677,9 @@ def make_default_bbr_category(self, cache):
             if self.kmb_info.category_exists(test_cat, cache):
                 self.content_cats.add(test_cat)
             else:
-                # @todo: add an entry to the log here
+                self.log.write(
+                    'Had to fall back on "Listed buildings in Sweden" for '
+                    'image with id "{}".'.format(self.ID))
```

Comment:
Maybe for log lines that include a file ID, put it right at the beginning? `[self.ID] -- Had to fall back etc`. Makes it easy to visually parse the log for problems related to the same ID, since they'll line up.